SPECS/gawk/gawk.spec

```diff
@@ -40,6 +40,8 @@ find %{buildroot}%{_libdir} -name '*.la' -delete
 # Skip the timeout test, which is unreliable on our (vm) build machines
 sed -i 's/ timeout / /' test/Makefile
 sed -i 's/ pty1 / /' test/Makefile
+# Skip localenl test, we do not support `en_US.ISO-8859-1`
```

Comment:
Are we sure about that? I've seen `ISO-8859-1` being mentioned in a few spec files. For instance check the `%prep` section inside `SPECS/pcsc-lite/pcsc-lite.spec`.

Reply:
Thanks for the review. Digging :)